knora/dsplib/schemas/data.xsd

```diff
@@ -413,6 +413,7 @@
         <xs:attribute name="iri" type="xs:string" use="optional"/>
         <xs:attribute name="permissions" type="xs:NCName" use="optional"/>
         <xs:attribute name="ark" type="xs:string" use="optional"/>
+        <xs:attribute name="creationDate" type="xs:dateTime" use="optional"/>
```

Comment:
But in the long run, it would probably make sense to upgrade to XSD 1.1 if the API uses it? Did you try to switch or did you "just" do theoretical research?

Reply:
actually, I think the docs may be wrong here:

![image](https://user-images.githubusercontent.com/33053745/195025332-e8f36d31-b0e5-446f-88f0-15102a4c2802.png)

I queried the prod-database for last modification dates, and they seem to be `xsd:dateTime`. (I have some vague memory that at some point we had a mix between the two in the database...)

Reply:
So, this means that both API and TOOLS only support XSD 1.0 (with `xsd:dateTime`). If we want to switch to XSD 1.1, this needs research by Balduin/Irina, so it is something for the future.  

For the moment, I suggest that I finish this PR with XSD 1.0.
And then, I would find it nice to make a PR in API that replaces all occurrences of `xsd:dateTimeStamp` by `xsd:dateTime`. There are more than 100 occurrences. But I'm unsure if it is really that easy. I just think that it's not proper to have this in the API if it is not really supported.  

What do you think?

Reply:
I found it with a repo-wide search in PyCharm. I didn't want to change data, just the API code (mostly comments) and docs

Reply:
that's dangerous because then it's possible to send a `xsd:dateTime` without timezone information - and what would that actually mean? GMT? GMT+2? ...?  
I think that's why we use `xsd:dateTimeStamp`, so that we can be sure it's always including the timezone, and then in the triplestore we can still use `xsd:dateTime`, but within the API ensure the timezone information is present

Reply:
Okay, but then my 100'000 $ question is: How can I use the https://www.w3.org/TR/xmlschema11-2 namespace?